playwright/test/selectors/item.ts

```diff
@@ -1,3 +1,10 @@
 export const zoomInButton = `[data-test-id="zoom-in-button"] button`;
+export const rotateButton = `[data-test-id="rotate-button"] button`;
 export const openseadragonCanvas = `.openseadragon-canvas`;
 export const fullscreenButton = `[data-test-id="fullscreen-button"] button`;
+export const searchWithinResultsHeader = `[data-test-id="results-header"]`;
+export const mainViewer = `[data-test-id=main-viewer] > div`;
+export const activeIndex = `[data-test-id=active-index]`;
+export const accordionItemLicense = `[data-test-id=accordion-item-license]`;
+export const accordionItemVolumes = `[data-test-id=accordion-item-volumes]`;
+export const accordionItemContents = `[data-test-id=accordion-item-contents]`;
```

Comment:
Is there a reason we're using so many test IDs rather than what the user sees?

Reply:
For some of these such I don't think the text has been set in stone, so using the data attribute is more robust. For others like the activeIndex it just seemed like the easiest way to grab the text without having to extract the relevant part from a string

Reply:
The point of _not_ using the selectors is so that your tests are brittle to changes, including text changes, that affect the people using the service. If we change the text, we could just change the test.

Do we agree that we should be using user facing attributes? I feel we haven't even tried and gone back to the more implementation based testing that I was hoping to avoid. Ping @davidpmccormick as it affects other devs on this.

Reply:
The data-test-id selector is ok though. The best practice guide says, "When user-facing attributes change frequently, it is recommended to use explicit test ids, like data-test-id."

Reply:
That's not how I read the https://playwright.dev/docs/selectors/#best-practices

Reply:
but I can see it what you're saying

Reply:
the best practice says, "To reduce the maintenance burden, we recommend prioritizing user-facing attributes and explicit contracts", because they  "change rarely" and "When user-facing attributes change frequently, it is recommended to use explicit test ids" because they don't. So the intention is to make the tests less brittle to change. 

But I can see the advantage of using the user facing attributes, so happy to go that way - it's not like we're going to be changing these things a lot

Reply:
My point is that we have resorted to the fallback before we have even proved that things are going to change regularly.